Gemfile

```diff
@@ -119,7 +119,7 @@ gem 'social-share-button'
 gem 'select2-rails', '~> 4.0'
 
 # follower functionality
-gem 'acts_as_follower', github: 'tcocca/acts_as_follower'
+gem 'acts_as_follower'
```

Comment:
I believe the version listed on Rubygems has an issue with Rails 5+, which is why we switched to using the source here

Reply:
Got it. Restoring github